### DISCUSSION TITLE
Update version

### DISCUSSION THREAD

**Discussion by kerryw92** [pdulvp/jellyfin-qnap#86] [General]
Hopefully someone can help, I initially installed Jellyfin 10.8.13 by manual install after downloading from here, my issue now is I realize if I added the app repo and installed that version I could easily press update to get the latest versions.

How would I go about updating my install to 10.9.1 while keeping all library settings, watch info etc?

Am I best to stop Jellyfin & copy off some config files/folders, remove the manually installed version and then re-install using the repo, once done I can copy back the backed up files to restore my settings etc?

Which files/folders would I need? Im guessing they are found here - /share/CACHEDEV1_DATA/.qpkg/jellyfin

Much appreciated!

**Comment by Oednerich** [comment]
Hey kerryw92,

you can just add the repo after manual qpkg installation and update from there, I was wondering the same so I made a backup (see this: https://github.com/pdulvp/jellyfin-qnap/discussions/80 for more info on the how-to-backup) and tried it.
you can also install/update through the repo and install the next release via qpkg file here from github, if the repo is too slow for you or you need the update asap because of a bugfix and it's not yet in the repo.
Someone please correct me if I'm wrong, but the qpkg file here and in the repo are the same, so it won't make a difference from where you update, jellyfin won't install a second instance on your NAS if you're fearing that.

10.9.2 has already been released here by pdulvp, so you should go for that. official jellyfin 10.9.3 has also been released, but its just some small bugfixes, didnt see anything major and its not here yet.

Just do a backup like any sane person would and nothing can go wrong ;)

**Reply by kerryw92** [reply]
Ah thank you for clarifying!

Only bit worrying me was the manual install shows in the usual qnap apps area however when I click the repo tab I have the option to install Jellyfin again but not update so wasn't sure how it would react and if it would know it was already installed.

I'll do a backup of the folders mentioned and go from there! Thanks so much.

Didn't fancy spending hours sorting metadata again haha. Will update this with me progress in a few days time to hopefully document for others who might have made the same mistake.

**Reply by Oednerich** [reply]
just for clarification, I updated that way (first qpkg, then add the repo and update from there) on my old NAS, but iirc the app will go to the pdulp repo after installing / updating. everything went fine, but just do a backup anyway.
After seeing other people with install / update issues that I personally never had, doing a backup beforehand is always the best option.
Every blogpost from official jellyfin is recommending a backup, just in case.

edit: the easiest way to save the folders imho is to just use winscp, just remember the filepath uses hidden folders ( /.qpkg/ ) so either copy/paste the full path to the jellyfin folder or check winscp settings and activate "show hidden files".
well, if your on windows anyway. just adding this for other people that want to do backup that way.

edit2: also, before the backup, make sure to go into the app center and stop the jellyfin app from running!

**Comment by ghost** [comment]
I manually installed the QPKG over the top of an existing installation, and then added your repo. 

When browsing your repo in QNAP, it show Jellyfin as not installed. 
It is showing as installed in another repo.

Fun times!
